Anon-Inu/poc-network
# Linting + prettying

**Description**
* Adds linting and prettying to the project
* Per [this thread](https://github.com/Anon-Inu/poc-network/issues/6#issuecomment-985476066) we're going to make all eslint issues warnings so commits aren't blocked by errors
* For anyone with `VSCode` there is a workplace config that allows you to run prettier on save

**Requirements**
* Husky requires > Git 2.9- you will need to update if you are running a lower version
* For VSCode users- install the `esbenp.prettier-vscode` extension

**To Test**
* Pull the branch and run `npm install`. Make a fake commit and hopefully you see the linter and prettier run.
* If you have `VSCode`, make some code changes and save. It should reformat
